packages/botonic-plugin-directus/src/cms/cms.ts

```diff
@@ -0,0 +1,21 @@
+import { Button, Text, Image } from './contents'
+
+export enum ContentType {
+  TEXT = 'text',
+  BUTTON = 'button',
+  IMAGE = 'image',
+}
```

Comment:
Now there are not that many contents, but in the future it may be useful to split them in `SubContentType` and `TopContentType` as we do in Contentful
```
export enum SubContentType {
  BUTTON = 'button',
  ELEMENT = 'element',
}

export type ContentType = TopContentType | SubContentType
export const ContentType = { ...TopContentType, ...SubContentType }
```